src/amp-story-player/amp-story-player-impl.js

```diff
@@ -211,6 +216,26 @@ export class AmpStoryPlayer {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /**
+   * Helper to create a button.
+   * @private
+   */
+  initializeButton_() {
+    const option = this.element_.getAttribute('button');
+    if (!Object.values(BUTTON_TYPES).includes(option)) {
+      return;
+    }
+
+    const button = this.doc_.createElement('a');
+    button.classList.add('amp-story-player-' + option + '-button');
```

Comment:
nit: I would have a slight preference for having a separate map for class names (perhaps keyed by the button type)

e.g.

```javascript
const BUTTON_CLASSES = {
  [BUTTON_TYPES.BACK]: 'amp-story-player-back-button',
  [BUTTON_TYPES.CLOSE]: 'amp-story-player-close-button',
};

/* ... */

button.classList.add(BUTTON_CLASSES[option]);
```

This way, you can:

1. See all the possible classnames up front
2. Reuse the same class name across multiple button types (if you ever wanted to)